airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java

```diff
@@ -408,6 +412,15 @@ private Job getJob(DSLContext ctx, long jobId) {
         .flatMap(r -> getJobOptional(ctx, r.get("job_id", Long.class))));
   }
 
+  @Override
+  public List<Job> listJobs(ConfigType configType, Instant attemptEndedAtTimestamp) throws IOException {
+    final LocalDateTime timeConvertedIntoLocalDateTime = LocalDateTime.ofInstant(attemptEndedAtTimestamp, ZoneOffset.UTC);
+    return database.query(ctx -> getJobsFromResult(ctx
+        .fetch(BASE_JOB_SELECT_AND_JOIN + "WHERE " +
+            "CAST(config_type AS VARCHAR) IN " + Sqls.toSqlInFragment(Sets.newHashSet(configType)) + " AND " +
+            " attempts.ended_at > ? ORDER BY jobs.created_at ASC, attempts.created_at ASC", timeConvertedIntoLocalDateTime)));
```

Comment:
```suggestion
        .fetch(BASE_JOB_SELECT_AND_JOIN + "WHERE " +
            "CAST(config_type AS VARCHAR) = ? AND attempts.ended_at > ? " +
            ORDER_BY_JOB_TIME_ATTEMPT_TIME, 
            configType, 
            timeConvertedIntoLocalDateTime)));
```

This may not be 100% syntactically right, but I think it expresses what I mean.